Makefile

```diff
@@ -8,6 +8,7 @@ AUTO_INSTRUMENTATION_PYTHON_VERSION ?= "$(shell grep -v '\#' versions.txt | grep
 AUTO_INSTRUMENTATION_DOTNET_VERSION ?= "$(shell grep -v '\#' versions.txt | grep aws-otel-dotnet-instrumentation | awk -F= '{print $$2}')"
 DCGM_EXPORTER_VERSION ?= "$(shell grep -v '\#' versions.txt | grep dcgm-exporter | awk -F= '{print $$2}')"
 NEURON_MONITOR_VERSION ?= "$(shell grep -v '\#' versions.txt | grep neuron-monitor | awk -F= '{print $$2}')"
+TARGETALLOCATOR_VERSION ?= "$(shell grep -v '\#' versions.txt | grep targetallocator |  awk -F= '{print $$2}')"
```

Comment:
Nitpick - should be ```target-allocator``` instead of ```targetallocator``` here and all the references

Reply:
To match the others, we can change it to that. The reason I used this is because upstream does that too: https://github.com/open-telemetry/opentelemetry-operator/blob/v0.90.0/Makefile#L7.